extensions/azurecore/src/account-provider/azureAccountProviderService.ts

```diff
@@ -70,8 +70,7 @@ export class AzureAccountProviderService implements vscode.Disposable {
 		// let self = this;
```

Comment:
While you're here mind cleaning this up too?

Reply:
Oh this is a port request. Doesn't really matter for this then.

Reply:
I'll clean up in my MSAL work